packages/web/app/src/components/ui/date-range-picker.tsx

```diff
@@ -231,6 +231,49 @@ export function DateRangePicker(props: DateRangePickerProps): JSX.Element {
     );
   };
 
+  // Dynamic presets
+  const [dynamicPresets, setDynamicPresets] = useState<Preset[]>([]);
+  useEffect(() => {
+    if (quickRangeFilter.trim().match(/^Last [0-9]+$/)) {
```

Comment:
maybe also make it case-insensitive? so `last` also works?

Reply:
we need to cover a, where somebody types `3` and gets `Last 3 hours / days / weeks` etc.

Basically a similar experience to the one in [Sentry](https://the-guild-z4.sentry.io/issues/?project=6716298&statsPeriod=30d&utc=true)